ibis/omniscidb/tests/test_client.py

```diff
@@ -177,10 +177,7 @@ def test_explain(con, alltypes):
 
 @pytest.mark.parametrize(
     'filename',
-    [
```

Comment:
why does this need a path at all?

Reply:
well, we need temp file on the server to where and from where we gonna write/read our data,
`read_csv()` can take `str` and `pathlib.Path`, so for the test we should pass filepath with two ways